src/js/routes/Welcome.jsx

```diff
@@ -60,6 +61,32 @@ export default class Intro extends Component {
     this.setState({loading: true});
   }
 
+  shareToFacebookButton () {
```

Comment:
These 2 buttons could be extracted out into social buttons component that could then make them more reusable around the app. Then, at the same time, if more social buttons are added, you won't have to go into files to add or update the buttons that are already in any files. Just a thought.